### PR TITLE
Update example blog application from guides

### PR DESCRIPTION
**Pull request by werdlerk**
Update the example blog application created in the [updated guides](https://github.com/zilverline/sequent/pull/424) on sequent.io

### CODE REVIEW

File: building-a-web-application/app/web.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 require 'sinatra/base'
-require 'sinatra/flash'
-require 'sinatra/reloader'
+require 'sinatra/flash' # for displaying flash messages
+require 'sinatra/reloader' # for hot reloading changes we make
```
Comment (lvonk) [inline building-a-web-application/app/web.rb:3]:
```suggestion
require 'sinatra/reloader' # for hot reloading changes
```

File: building-a-web-application/lib/usernames/usernames.rb [hunk 2]
```diff
@@ -5,7 +5,7 @@ class UsernameAlreadyRegistered < StandardError; end
   ID = "85507d60-8645-4a8a-bdb8-3a9c86a0c635"
 
   def self.instance(id = ID)
-    Sequent.aggregate_repository.load_aggregate(id)
+    Sequent.configuration.aggregate_repository.load_aggregate(id)
```
Comment (lvonk) [inline building-a-web-application/lib/usernames/usernames.rb:8]:
```suggestion
    Sequent.aggregate_repository.load_aggregate(id)
```

File: building-a-web-application/spec/spec_helper.rb [hunk 3]
```diff
@@ -8,7 +8,12 @@
 
 require_relative '../blog'
 
-Sequent::Test::DatabaseHelpers.maintain_test_database_schema(env: 'test')
```
Comment (lvonk) [inline building-a-web-application/spec/spec_helper.rb:11]: Why not keep this actually?

File: building-a-web-application/lib/usernames/usernames.rb (hunk 2 above)
Comment (lvonk) [reply building-a-web-application/lib/usernames/usernames.rb:8]: Use the shortcuts

File: building-a-web-application/spec/spec_helper.rb (hunk 3 above)
Comment (werdlerk) [reply building-a-web-application/spec/spec_helper.rb:11]: It's currently not in the [generated spec_helper.rb](https://github.com/zilverline/sequent/blob/master/lib/sequent/generator/template_project/spec/spec_helper.rb) which is why this was removed/replaced. 

Should I add this change to the generated `spec_helper.rb` instead?

Comment (lvonk) [reply building-a-web-application/spec/spec_helper.rb:11]: Yes, I think we forgot to update that one.

**Review by lvonk — COMMENTED** [review]
Looks good, but we should change the spec_helper

File: building-a-web-application/spec/spec_helper.rb (hunk 3 above)
Comment (werdlerk) [reply building-a-web-application/spec/spec_helper.rb:11]: [Updated the spec_helper.rb in the Sequent project generator](https://github.com/zilverline/sequent/commit/920bfc42359c96d483b03950250e545151907e8d).

**Comment by werdlerk** [comment]
I think this can be merged. Agree to merge, @lvonk ?

**Pull request merged**